docs/root/operations/admin.rst

```diff
@@ -354,6 +354,22 @@ modify different aspects of the server:
   Envoy has updated (counters incremented at least once, gauges changed at least once,
   and histograms added to at least once)
 
+  .. http:get:: /stats?recentlookups
```

Comment:
I don't think this path is right, also can you doc all the paths? Also can you add a release note?

Reply:
I'm still tweaking the doc a bit. I'll re-add 'waiting' tag until it's ready.

Reply:
ok; doc fixed, release note added.